frontend/package.json

```diff
@@ -32,15 +34,5 @@
     "not dead",
     "not ie <= 8",
     "not op_mini all"
-  ],
-  "devDependencies": {
-    "eslint": "5.6.0",
```

Comment:
I don't understand why we are removing these. They provide important errors/warnings that don't show up in our editor, and allow us to not use the same editor and get the same warnings.

Reply:
Give me a couple minutes and I'll send you the warnings/errors that occur when I leave them in

Reply:
with all the devdeps in place:
![image](https://user-images.githubusercontent.com/2746350/53660609-437cf100-3c2c-11e9-9342-17b6e91a9ba6.png)
![image](https://user-images.githubusercontent.com/2746350/53660629-50014980-3c2c-11e9-9c04-a727f361991d.png)